Backend/Remora.Discord.API/API/Formatting/Markdown.cs

```diff
@@ -0,0 +1,119 @@
+//
+//  Markdown.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using JetBrains.Annotations;
+
+namespace Remora.Discord.API.Formatting
+{
+    /// <summary>
+    /// Provides helper methods to format strings into Discord Markdown.
+    /// </summary>
+    [PublicAPI]
+    public static partial class Markdown
```

Comment:
How about if the content already contains certain markdown characters - perhaps escaping those is a good idea